reproject/mosaicking/coadd.py

```diff
@@ -165,6 +188,9 @@ def reproject_and_coadd(
 
         shape_out_indiv = (jmax - jmin, imax - imin)
 
+        if block_sizes is not None:
+            kwargs['block_size'] = block_sizes[idata]
```

Comment:
It would be nice to allow ``block_sizes`` to perhaps be a single shape that would be the same for all images. To some extent, in the dask case (once implemented) the block size won't need to be passed in as it will be inferred from the dask arrays themselves so at that point maybe the single block size input would actually become the most commonly used.

Reply:
right, this might be good to refactor but I'll make it flexible for now